crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -50,6 +50,9 @@ spec:
       username_claim: "sub"
     openshift:
       client_id_prefix: "kiali"
+      custom_ca: ""
+      server_prefix: "https://kubernetes.default.svc/"
+      use_system_ca: false
```

Comment:
Do we need "use_system_ca"? Could we use the "custom_ca" field for both? By that I mean, if "custom_ca" is empty string (the default), wouldn't that assume you want to use the system CA (use_system_ca=true)? And if "custom_ca" is not an empty string, wouldn't that mean you are NOT using the system CA (i.e. use_system_ca=false)??

In other words: `use_system_ca = (custom_ca == "")`

Reply:
Yep, we need it because this now becomes a three way switch. The third case is the existing default behavior, when the CA is fetched from the local kubernetes injected file next to the API token and other stuff.
System CA however means kiali is using the public trusted CA certificate bundle packaged with the base image of the container.